can/CANMsgMap.cpp

```diff
@@ -0,0 +1,55 @@
+#include "CANMsgMap.h"
```

Comment:
Edit: changed it so that one-shot RX messages are also put in the CANMsgsMaps. This just offloads the message unpacking logic to the CAN library instead of the app, to make the application code more readable